configuration/configuration.go

```diff
@@ -153,27 +153,27 @@ func IsPostgresDeveloperModeEnabled() bool {
 	return viper.GetBool(varDeveloperModeEnabled)
 }
 
-func getRSAPrivateKey() string {
-	return viper.GetString(varRSAPrivateKey)
+func GetDefaultPrivateKey() string {
```

Comment:
You are fast ! Thanks, added in https://github.com/almighty/almighty-core/pull/337/files#diff-786c8376feaa68e9ebee88ca950d2774R156